### PR TITLE
[DIG2022-580] Adding new campaign type

### PR DESCRIPTION
**Pull request by shalingams**
## Description
Adding tactic as a argument to report request create method.

### JiraID
[DIG2022-580](https://digivizer.atlassian.net/browse/DIG2022-580)

### CODE REVIEW

File: lib/blurb/report_requests.rb [hunk 1]
```diff
@@ -2,29 +2,31 @@
 
 class Blurb
   class ReportRequests < RequestCollectionWithCampaignType
-    SD_TACTIC = 'T00020'.freeze
 
     def initialize(campaign_type:, base_url:, headers:)
       @campaign_type = campaign_type
       @base_url = "#{base_url}/v2/#{@campaign_type}"
       @headers = headers
+      @tactic = "T00020"
```
Comment (rrails) [inline lib/blurb/report_requests.rb:10]: We need to pass it as a parameter not hardcode it to "T00020".

Comment (jziegel) [reply lib/blurb/report_requests.rb:10]: @rrails I think `T00020` should be the default for backwards compatibility and we use the passed `tactic` param if present

Comment (shalingams) [reply lib/blurb/report_requests.rb:10]: It get pass in to create method, in initialize I set it to a default value.

Comment (rrails) [reply lib/blurb/report_requests.rb:10]: Use param if present. So we should add the param now?

**Review by rrails — APPROVED** [review]

File: lib/blurb/report_requests.rb (hunk 1 above)
Comment (rrails) [reply lib/blurb/report_requests.rb:10]: I now understand - all good to go.

**Pull request merged**